### DISCUSSION TITLE
Will larger runners support public repositories?

### DISCUSSION THREAD

**Discussion by vgeddes** [community/community#38164] [Actions]
Currently public repositories are limited in what they can do with GitHub Actions, primarily because one cannot build workflows requiring a large amount of hardware resources. Self-hosted runners are not an option as they are not designed to work with public runners.

So will larger runners support public repositories, and will this support be available on Team plans?

I tried the larger runners beta program, but the default runner group excludes public repositories.

**Comment by ossanna16** [comment]
Hi there @vgeddes and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by mattjohnsonpint** [comment]
This is really important to understand.  I do see that you can enable public repositories:

<img width="1644" alt="image" src="https://user-images.githubusercontent.com/1396388/222298701-d53c2cc4-9fb3-4b9e-aed0-9425140ee697.png">

But it includes the warning about it being a major security risk for _self hosted_ runners.  I presume this is not applicable for the GitHub-hosted larger runners? The docs at  https://docs.github.com/en/actions/hosting-your-own-runners/about-self-hosted-runners#self-hosted-runner-security say:

> _This is not an issue with GitHub-hosted runners because each GitHub-hosted runner is always a clean isolated virtual machine, and it is destroyed at the end of the job execution._

So I presume that larger GitHub-hosted runners are also safe?  If so, why have the security warning checkbox?  Perhaps the UI has just not been updated for the beta?
